types/src/front/lib/assistant.ts

```diff
@@ -84,8 +84,7 @@ export function getLargeWhitelistedModel(
 
 export const GPT_3_5_TURBO_MODEL_ID = "gpt-3.5-turbo" as const;
 export const GPT_4_TURBO_MODEL_ID = "gpt-4-turbo" as const;
-export const GPT_4O_LEGACY_MODEL_ID = "gpt-4o" as const;
-export const GPT_4O_MODEL_ID = "gpt-4o-2024-08-06" as const;
```

Comment:
I thiiiink we will need to keep the version model as legacy to render old conversations?

Reply:
I did not see any issue when rendering the conversations, or when looking at dustapps runs (old name still showing even if we don't have the config anymore), but maybe safer to keep it  ?

Reply:
(just restored as GPT_4O_20240806_MODEL_ID in case we want to keep it)

Reply:
Dust apps run on your key and let you use all your models including fine tuned ones 👍

Reply:
I believe we need the model config to be able to run assistants whose model id is the model id we are removing before we migrate them. And then we can remove it right?

Reply:
Yes but we can migrate all assistants before merging ? or should we do this in 2 steps ?

Reply:
you always have a risk of a race but yeah you can migrate merge migrate and avoid 2 PRs :+1: